package.json

```diff
@@ -9,6 +9,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "argparse": "^2.0.1",
```

Comment:
I would like to keep this project so small and simple as possible, I think that the cli for it is not too complex for necessary add a external library to handle that. Could you do the parse of arguments no using libraries?